mkt/developers/models.py

```diff
@@ -193,41 +191,6 @@ class Meta:
         db_table = 'user_inapp_keys'
 
 
-class PreloadTestPlan(ModelBase):
-    addon = models.ForeignKey('webapps.Webapp')
```

Comment:
So, we are going to need a migration to remove old instances of this model as well, otherwise the FK is going to mess with us since Django won't know about it anymore.